### ISSUE TITLE
BUG: wrong order of keywords on creating transient database

### ISSUE THREAD

**Issue by aleenprd** [Titan-Systems/titan#97]
```
titan_test_db = Database(
    name="TITAN_TEST_DB",
    comment="This is a database used for testing of titan[core].",
    transient=True,
    owner="SYSADMIN",
    data_retention_time_in_days=None,
    max_data_extension_time_in_days=None,
)

```

results in 

`snowflake.connector.errors.ProgrammingError: 001003: failed to execute sql, [CREATE DATABASE TITAN_TEST_DB TRANSIENT COMMENT = $$This is a database used for testing of titan[core].$$]`

The correct syntax is:
`CREATE TRANSIENT DATABASE TITAN_TEST_DB  COMMENT = $$This is a database used for testing of titan[core].$$;`

**Comment by aleenprd** [comment]
Same thing for schemas` snowflake.connector.errors.ProgrammingError: 001003: failed to execute sql, [CREATE SCHEMA TITAN_TEST_DB.TITAN_TEST_SCHEMA TRANSIENT COMMENT = $$This is a schema used for testing purposes of titan[core].$$]`

**Comment by teej** [comment]
Fixed in [v0.9](https://github.com/Titan-Systems/titan/releases/tag/v0.9.1)
